velox/functions/prestosql/window/tests/WindowTestBase.h

```diff
@@ -128,7 +155,8 @@ class WindowTestBase : public exec::test::OperatorTestBase {
       const std::vector<RowVectorPtr>& input,
       const std::string& function,
       const std::string& overClause,
-      const std::string& errorMessage);
+      const std::string& errorMessage,
+      const std::optional<std::string> frameClause);
```

Comment:
Please add the frameClause as a default value, so that you needn't change function signatures where frame testing is not required.

Reply:
Updated, thanks!